content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -281,8 +281,8 @@ const getPostsAndCategories = await prisma.post.findMany({
 Implicit m-n relations:
 
 - Use a specific [convention for relation tables](#conventions-for-relation-tables-in-implicit-m-n-relations)
-- Do **not** require the `@relation` attribute unless you need to [disambiguate relations](/concepts/components/prisma-schema/relations#disambiguating-relations) (this is not specific to implicit n-m relations).
-- If they use an `@relation` attribute for disambiguation, you can still not use the `fields` and `references` arguments as these are implicit and fixed for implicit m-n relations. 
+- Do **not** require the `@relation` attribute unless you need to [disambiguate relations](/concepts/components/prisma-schema/relations#disambiguating-relations) with a name, e.g. `@relation("MyRelationTable")` or `@relation(name: "MyRelationTable")`.
+  - If you do use the `@relation` attribute for disambiguation, you cannot use the `references`, `fields`, `onUpdate` or `onDelete` arguments. This is because these take a fixed value for implicit m-n relations and cannot be changed.
```

Comment:
You can never use the other arguments, not only if you use `@relation` for disambiguation.

Reply:
This is still unclear unfortunately and makes it sounds like this is a special case of when you disambiguate. But this is always the case.